lib/Dialect/FIRRTL/Transforms/PrefixModules.cpp

```diff
@@ -173,6 +176,23 @@ void PrefixModulesPass::renameModuleBody(std::string prefix, FModuleOp module) {
   // on the instance.
   removeDeadAnnotations(thisMod, module);
 
+  mlir::AttrTypeReplacer replacer;
+  replacer.addReplacement(
```

Comment:
Neat, didn't know about this!

Reply:
Yeah, always great to find MLIR APIs with no public examples of their use!